mentii/mentii
# Select template

We can now choose a problem template based off the users history. 

**Significant Changes**
1. A users history is stored within a book section 
2. There are some functions in problem_ctrl that deal with the math related to picking the problem 
3. There is an edit book function that I use to update the book with the user's data

**How to Test**
1. Login
2. Do a Problem
3. Check the database to see if your name was added in the section you were working in. 

**Notes**
The only way to test this story is to check the database. The index of the problem you were working on will be incremented by one when you click the "return to class" button after successfully completing a problem. The same index will be decremented by 1 when you reach the end of a bad step path.
